### DISCUSSION TITLE
image2dxf

### DISCUSSION THREAD

**Discussion by gkadntmd** [mozman/ezdxf#1199] [Q&A]
Is there a way to create a dxf file using the qr code image as input?

**Comment by mbway** [comment]
There is no built-in mechanism. You would have to implement this yourself

**Comment by gkadntmd** [comment]
I binarized the qr image using opencv and converted it to dxf using lwpolyline, but when I actually laser engraved it, it came out blurry. Is there a way to make it print more clearly?
